exercises/02_check_palindrom/palindrom_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import "testing"
+
+func TestIsPalindrom(t *testing.T) {
+	for _, tc := range []struct {
+		name  string
+		input string
+		want  bool
+	}{
+		{"empty", "", true},
+		{"one word", "abc", false},
+		{"palindrom letters", "aaa", true},
+		{"palindrom numbers", "242", true},
+		{"one letter", "a", true},
+		{"special letter", "ū", true},
+		{"spaces", "   abc  ", false},
```

Comment:
nit: this function shouldn't care about spaces. meaning these are just regular symbols. so tests with and without spaces exist only for some kind of a demonstration purpose